### DISCUSSION TITLE
Sonoff 4CH PRO R3 - 4th channel switching on/off itself

### DISCUSSION THREAD

**Discussion by mario-website** [RavenSystem/esp-homekit-devices#1636] [General]
Hi everyone!

Channel number 4 on device switching on/off and off/on itself and time between changing is very randomly from 1 sec to few minutes. Factory reset not helped. I managed to use some combination of factory reset and managed to stop doing at all like it should be but after powered off and on, problem came back. It looks like ot problem is with remote control frequency but I’m not in 100% sure. 
I tried to do before flashing and after flashing - no luck.

Thanks in advance!

**Comment by mariusvasss** [comment]
Turn off radio! Interference

**Reply by mario-website** [reply]
There is no option to turn on or off. Only one way how I fixed is to assign to remote control that channel who switching itself to remote control and then remove pairing.

**Reply by shadowgrapher** [reply]
The Sonoff CH4 PRO is subject to radio frequency disturbances with random activation of the buttons. To eliminate interference due to the U7 (SYN470RU7 RF-433 module) you can cut the wire between U7 and U8 (MCU STM32f030c6). This line starts from pin 10 of U7, the part towards MCU can be soldered to ground (surrounding area). It is also possible to set the RF module (U7) in shutdown-mode by unsoldering or cutting pin 11.
![Disabilitare RF ch4_pro](https://user-images.githubusercontent.com/10960863/219968252-9f734ac7-1e22-448e-a20b-f96c3254d922.jpg)
